apps/predbat/predbat.py

```diff
@@ -7712,6 +7718,10 @@ def calculate_plan(self, recompute=True):
                 save="best",
                 end_record=self.end_record,
             )
+            # round charge_limit_best (kWh) to 1 decimal place and discharge_limits_best (percentage) to nearest whole number  gc
+            self.charge_limit_best = [ self.dp1(elem) for elem in self.charge_limit_best ]
```

Comment:
I'm not 100% sure 1 decimal place is enough here as it needs to be converted back into the right percentage later. I wonder if you are trying to fix the formatting of the text rather than really wanting to modified the save values that will be used to program the inverter?